### PR TITLE
AOs per mode

### PR DESCRIPTION
**Pull request by tarcisiozf**
* send mode on `data.ao`
* send mode on `data.ao.stopped`
* send mode on `data.aos`

### CODE REVIEW

File: lib/ws_servers/api/snapshots/send_all.js [hunk 1]
```diff
@@ -14,5 +13,4 @@ module.exports = async (session, ws) => {
   await sendUserInfo(session, ws, rest)
   await sendWallets(session, ws, rest)
   await sendPositions(session, ws, rest)
-  await sendOrders(session, ws, rest)
```
Comment (avsek477) [inline lib/ws_servers/api/snapshots/send_all.js:17]: how does frontend show the active orders if we don't send from the backend?

**Review by avsek477 — CHANGES_REQUESTED** [review]
updating the tests is needed as well.. rest lgtm

File: lib/ws_servers/api/snapshots/send_all.js (hunk 1 above)
Comment (tarcisiozf) [reply lib/ws_servers/api/snapshots/send_all.js:17]: the FE will not flush the data anymore, so it keeps track of the AOs from both modes

Comment (tarcisiozf) [reply lib/ws_servers/api/snapshots/send_all.js:17]: UPDATE: we actually need that, but in a different form, I'll move this task to draft again

**Pull request merged**